README.md

```diff
@@ -210,6 +246,7 @@ build-time configuration, environment variables, and expected artifacts.
 
 ---
 
+<!-- deprecate devkit -->
```

Comment:
What's the plan for this? Why not just add a depreciation note rather than a header?